templates/mod-box.html

```diff
@@ -2,7 +2,11 @@
 <div class="flip-container" ontouchstart="this.classList.toggle('hover');">
 	<div class="changer">
     <div class="thumbnail front" id="modbox-{{ mod.id }}-pic" data-modid="{{ mod.id }}" style="width: 100%;">
-        <div class="ksp-update">For {{ mod.default_version.gameversion.friendly_version }}</div>
+        <div class="ksp-update">For
+        {% if mod.default_version %}
```

Comment:
I think this `if` block could be applied to the whole div.

Reply:
Yeah, that's probably a better solution. Just hide the entire "For" box if we don't know it.